src/mjolnir/pbfgraphparser.cc

```diff
@@ -867,20 +871,23 @@ struct graph_callback : public OSMPBF::Callback {
     }
 
     for (const auto& tag : *results) {
-      if (tag.first == "iso:3166_1") {
-        bool hasTag = (tag.second.length() ? true : false);
-        if (hasTag) {
-          // Add the country iso code to the unique node names list and store its index in the OSM
-          // node
-          n.set_country_iso_index(osmdata_.node_names.index(tag.second));
-          ++osmdata_.node_name_count;
-        }
-      } else if ((tag.first == "state_iso_code")) {
-        bool hasTag = (tag.second.length() ? true : false);
-        if (hasTag) {
-          // Add the state iso code to the unique node names list and store its index in the OSM node
-          n.set_state_iso_index(osmdata_.node_names.index(tag.second));
-          ++osmdata_.node_name_count;
+      if (!use_admin_db_) {
+        if (tag.first == "iso:3166_1") {
```

Comment:
I think it would be cleaner to do if (tag.first == "iso:3166_1" && !use_admin_db_)